src/plugins/data_views/common/utils.test.ts

```diff
@@ -7,43 +7,47 @@
  */
 
 import { isFilterable } from '.';
-import { IFieldType } from './fields';
+import { DataViewField } from './fields';
```

Comment:
```suggestion
import type { DataViewField } from './fields';
```
There are a few more examples of this type being imported without `type`, while others are using `import type`. Do you think it would be worth it to use `import type` on all of them?